azext_iot/tests/test_iot_central_int.py

```diff
@@ -220,6 +220,20 @@ def test_central_device_methods_CRD(self):
 
         self._delete_device(device_id)
 
+    def test_central_user_methods_CRD(self):
+        (user_id, email) = self._create_user()
```

Comment:
Only the admin role is being tested here right?  What about tests for creating users in other roles?